src/components/flyout/flyout.spec.tsx

```diff
@@ -95,5 +127,23 @@ describe('EuiFlyout', () => {
           expect(cy.get('[data-test-subj="flyoutSpec"]').should('exist'));
         });
     });
+
+    it('does not close the flyout when the toast is clicked when `ownFocus=true`', () => {
+      cy.mount(<FlyoutWithToasts />);
+      cy.get('[data-test-subj="toastCloseButton"]')
+        .realClick()
+        .then(() => {
+          expect(cy.get('[data-test-subj="flyoutSpec"]').should('exist'));
+        });
+    });
+
+    it('closes the flyout when the toast is clicked when `ownFocus=false`', () => {
+      cy.mount(<FlyoutWithToasts ownFocus={false} outsideClickCloses={true} />);
```

Comment:
Wait, based on my understanding on the logic flow below, it's not actually `ownFocus={false}` that's making the flyout close, it's `outsideClickCloses={true}`. Am I not reading that correctly? Shouldn't this be

```suggestion
    it('closes the flyout when the toast is clicked when `outsideClickCloses=true`', () => {
      cy.mount(<FlyoutWithToasts outsideClickCloses={true} />);
```

Reply:
I'd also like to see 1 last test added for `outsideClickCloses={false}` and clicking on the overlay (also based on my understanding of the conditional branching below)